src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -361,7 +361,7 @@ export const VaultDetailPanels = ({
           </ChartValueContainer>
 
           <StyledLineChart
-            chartData={selectedUnderlyingData ? chartData?.underlying : chartData?.usd}
+            chartData={selectedUnderlyingData ? chartData?.underlying ?? [] : chartData?.usd ?? []}
```

Comment:
Instead of guarding this to never be undefined, shouldnt we guard it or fix it on the file/line it was exploding? Because from the `VaultDetailPanelsProps` interface we say that `chartData` could be undefined still.
So this line fixes this specific bug in this specific file but will not prevent for the bug to reappear if we use the LineChart component somewhere else and pass chartData as undefined from there, right?